include/knowhere/comp/brute_force.h

```diff
@@ -20,14 +20,15 @@ namespace knowhere {
 class BruteForce {
  public:
     static expected<DataSetPtr, Status>
-    Search(const DataSetPtr base_dataset, const DataSetPtr query_dataset, const Json& config, const BitsetView& bitset);
+    Search(const DataSetPtr base_dataset, const DataSetPtr query_dataset, const Json& json, const BitsetView& bitset);
```

Comment:
Let's keep it config. Its type is Json but role here is config, so its variable name should be config.